services/core-api/app/api/mines/mine/resources/mine_type.py

```diff
@@ -39,8 +41,9 @@ def post(self, mine_guid):
         mine_tenure_type_code = data['mine_tenure_type_code']
         mine_disturbance_code = data['mine_disturbance_code'] or []
         mine_commodity_code = data['mine_commodity_code'] or []
+        permit_guid = data['permit_guid']
```

Comment:
What will happen here when `permit_guid` is not provided? Can't some mine types, i.e. those ones belonging to a government agency, be created without a permit?

Reply:
correct. permit_guid is not required. Mine types can be created with the mine_guid only.

Reply:
discussed on a call